modules/monitoring-configuring-a-persistent-volume-claim.adoc

```diff
@@ -52,18 +48,21 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
-    <component>:
+    <component>: #<1>
       volumeClaimTemplate:
         spec:
-          storageClassName: <storage_class>
+          storageClassName: <storage_class> #<2>
           resources:
             requests:
-              storage: <amount_of_storage>
+              storage: <amount_of_storage> #<3>
 ----
+<1> Specify the core monitoring component for which you want to configure the PVC.
+<2> Specify an existing storage class. If you do not specify any, the default storage class is used.
```

Comment:
Rephrasing suggestion:

<2> Specify an existing storage class. If a storage class is not specified, the default storage class is used.